src/App.js

```diff
@@ -1,19 +1,66 @@
 import React from 'react';
 import './App.css';
-
+import NewToDo from './components/NewToDo';
+import todosFromServer from './api/todos';
 import users from './api/users';
+import TodoList from './components/TodoList';
+
+class App extends React.Component {
+  initialTodos = todosFromServer.map((item) => {
+    const { id, title, completed } = item;
+
+    return {
+      ...{
+        id, title, completed,
+      },
+      userName: users.find(user => (user.id === item.userId)).name,
+    };
+  })
+
+  state = {
+    todos: this.initialTodos,
+    usersNames: users.map(user => user.name).sort(),
```

Comment:
```suggestion
    usersNames: [...users.map(user => user.name)].sort(),
```
sort mutates the initial array